frontstage/views/secure_messaging/create_message.py

```diff
@@ -16,26 +16,25 @@
 @secure_message_bp.route('/create-message/', methods=['GET', 'POST'])
 @jwt_authorization(request)
 def create_message(session):
-    case_id = request.args.get('case_id')
     survey = request.args['survey']
     ru_ref = request.args['ru_ref']
     party_id = session['party_id']
     form = SecureMessagingForm(request.form)
     if request.method == 'POST' and form.validate():
         logger.info("Form validation successful", party_id=party_id)
-        sent_message = send_message(party_id, case_id, survey, ru_ref)
+        sent_message = send_message(party_id, survey, ru_ref)
         thread_url = url_for("secure_message_bp.view_conversation",
                              thread_id=sent_message['thread_id']) + "#latest-message"
         flash(Markup('Message sent. <a href={}>View Message</a>'.format(thread_url)))
         return redirect(url_for('secure_message_bp.view_conversation_list'))
 
     else:
         return render_template('secure-messages/secure-messages-view.html',
-                               ru_ref=ru_ref, survey=survey, case_id=case_id,
```

Comment:
Do we not need this `case_id` in the secure message view anymore?

Reply:
No Secure message no longer sends a case event this is all it was used for.